composer.json

```diff
@@ -26,14 +26,14 @@
     }
   ],
   "require": {
-    "php": ">=7.2 <8.2",
+    "php": ">=7.2 <8.3",
     "ext-curl": "*",
     "ext-json": "*",
-    "apimatic/core-interfaces": "^0.1.0"
+    "apimatic/core-interfaces": "dev-3-php-8.2-support"
```

Comment:
This change was also intended for testing. Please use the correct core-interfaces version that we are going to release after PHP 8.2 support